### DISCUSSION TITLE
Tracking time in Projects<title>

### DISCUSSION THREAD

**Discussion by x24bvitaldigital** [community/community#75462] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi There, I am new to GitHub but greatly appreciate the tool so far!  Has anybody used the project tool to track time?  We bill based on time and was thinking it'd be nice to have it all in one spot.  Thank You.

**Comment by ivnvxd** [comment]
GitHub's project management tool is primarily designed for tracking issues, pull requests, and organizing work but isn't inherently built for time tracking.

For time tracking, you might need to integrate GitHub with third-party time tracking tools like [Toggl](https://toggl.com/), [Harvest](https://www.getharvest.com/), and [Clockify](https://clockify.me/). These tools often offer browser extensions or integrations that link directly with GitHub, enabling you to start a timer for an issue or a pull request right from GitHub.
